spec/lib/discourse_redis_spec.rb

```diff
@@ -23,18 +23,17 @@
       it 'should append namespace to the keys' do
         raw_redis.set('default:key', 1)
         raw_redis.set('test:key2', 1)
+        raw_redis.set('default:key3', 1)
 
         expect(redis.keys).to include('key')
         expect(redis.keys).to_not include('key2')
-        expect(redis.scan_each.to_a).to eq(['key'])
+        expect(redis.scan_each.to_a).to eq(['key', 'key3'])
```

Comment:
```suggestion
        expect(redis.scan_each.to_a).to contain_exactly('key', 'key3')
```

(CI is failing at this line)

Reply:
Thanks! sorted